views/js/admin/src/components/common/form/SelectFormGroup.vue

```diff
@@ -21,11 +21,11 @@ import { labelProps } from '@/composables/props/labelProps';
 import { useOptionsProps } from '@/composables/props/useOptionsProps';
 import { useSelectModel } from '@/composables/props/model/useSelectModel';
 
-const { model, props, setup } = useSelectModel();
+const {model, props, setup} = useSelectModel();
```

Comment:
Het maakt mij persoonlijk niet heel veel uit, maar bij een aantal van deze constructies zijn de omringende spaties verwijderd, en bij een aantal niet. Ik vind het mooier met spaties. Maar als de linter zegt dat het zonder moet, dan verwacht ik dat het overal waar het aangeraakt is zonder spaties is.